modules/ROOT/pages/1.5_Codes_and_Identifiers.adoc

```diff
@@ -146,14 +146,14 @@ This fragment of XML shows how the compulsory attributes are used for the some o
 . The UUID follows the ISO/IEC Scheme 9834-8:2008 Version 4 and was generated by the European Commission's Directorate General GROWTH (DG GROW)
 . The reference number used to identify to which procurement procedure this ESPD document belongs (PP.20170419.1024-9) has been supplied by the Spanish Directorate DGPE
 . Generic information, such as the content version ID, use always by default the "EU-COM-GROW" Agnecy ID. Notice that the other additional attributes may be also used, as in this example.
-. Beware that the codes may be numbers, text or combinations of both. These code labels are the ones that are specified in the codelist spreadsheets and XML Genericode files distributed jointly with this specification (in the folder /codelists of the link:{url-tree}/dist[distribution package].
+. Beware that the codes may be numbers, text or combinations of both. These code labels are the ones that are specified in the codelist spreadsheets and XML Genericode files distributed jointly with this specification (in the folder /codelists of the link:{url-tree-tmp}/dist[distribution package].
```

Comment:
point 6, link:{url-tree-tmp}/dist[distribution package does not work

Reply:
Fixed.